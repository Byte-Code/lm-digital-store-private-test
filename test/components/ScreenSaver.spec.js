import React from 'react';
import { shallow } from 'enzyme';

import ScreenSaver from '../../app/components/ScreenSaver';

describe('ScreenSaver', () => {
  it('should render properly', () => {
    const result = shallow(
      <ScreenSaver />
    );
    expect(result).toMatchSnapshot();
  });
});
