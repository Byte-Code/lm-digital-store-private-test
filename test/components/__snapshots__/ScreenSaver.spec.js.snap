// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScreenSaver should render properly 1`] = `
<video
  autoPlay={true}
  className="video"
  loop={true}
  muted={true}
>
  <source
    src="test-file-stub"
    type="video/mp4"
  />
</video>
`;
